packages/xo-web/src/common/xo/copy-vm-modal/index.js

```diff
@@ -2,23 +2,24 @@ import BaseComponent from 'base-component'
 import React from 'react'
 
 import _, { messages } from '../../intl'
+import SelectCompression from '../../select-compression'
 import SingleLineRow from '../../single-line-row'
 import Upgrade from 'xoa-upgrade'
 import { Col } from '../../grid'
 import { SelectSr } from '../../select-objects'
-import { Toggle } from '../../form'
 import { injectIntl } from 'react-intl'
 
 class CopyVmModalBody extends BaseComponent {
   state = {
-    compress: false,
+    compression: '',
     copyMode: 'fullCopy',
   }
 
   get value() {
     const { props, state } = this
     return {
-      compress: state.compress,
+      compress:
+        state.compression === 'zstd' ? 'zstd' : state.compression === 'native',
```

Comment:
Isn't it weird that it can either be a string or a boolean? I'm sure there's a good reason for that but I'd like to hear it :)

Reply:
For not break the API.